src/Ajax.php

```diff
@@ -266,19 +283,27 @@ public static function updateShipment() {
 
             // Update shipment details in the database.
             $shipment = Helper::saveShipmentWithJoinedItems($shipment);
+            $order->add_order_note( sprintf( __( 'Shipment %s was updated', 'trackmage' ), $shipment['tracking_number']), false, true );
 
             try {
                 // Get HTML to return.
                 ob_start();
                 include TRACKMAGE_VIEWS_DIR . 'meta-boxes/order-shipments.php';
                 $html = ob_get_clean();
+
+                ob_start();
+                $notes = wc_get_order_notes( array( 'order_id' => $orderId ) );
+                include WC()->plugin_path().'/includes/admin/meta-boxes/views/html-order-notes.php';
+                $notes_html = ob_get_clean();
+
             } catch (\Exception $e) {
                 wp_send_json_error(['message' => $e->getMessage()]);
             }
 
             wp_send_json_success([
                 'message' => __('Shipment updated successfully!', 'trackmage'),
                 'html' => $html,
+                'notes' => $notes_html
             ]);
         } catch (\Exception $e) {
             wp_send_json_error(['message' => $e->getMessage()]);
```

Comment:
Please remove the inner catch.
The nested catches logic is broken. If an error happens inside the inner catch, it will not stop the execution and `html, notes_html` variables will not be initialized.
![image](https://user-images.githubusercontent.com/1675033/68879732-904d1d00-0712-11ea-9254-fcf5b44be0c2.png)